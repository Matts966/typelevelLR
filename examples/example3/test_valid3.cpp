
#include <iostream>
#include "example3.hpp"
using namespace example3;

int main() {
  // 105 = 0b1101001
  std::cout << *begin()->I()->I()->O()->I()->O()->O()->I()->end() << std::endl;
  return 0;
}
