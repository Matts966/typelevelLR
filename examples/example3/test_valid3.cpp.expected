RuleS1(Rule11(Rule30(Rule61(Rule60(Rule50(Rule31(Accept())))))))